normalize_data.gv

```diff
@@ -0,0 +1,171 @@
+
+/* Start of top-level graph */
+digraph Workflow {
```

Comment:
(Since I can't leave a comment on the actual diagram) - please update the `README.md` to reference your new workflow and diagram, like the other sections.